README.md

```diff
@@ -104,6 +104,7 @@ Here is a nonexhaustive list of boards and modules that we have tested with the
 - LTEK_FF1705.
 - Advantech Wise 1510.
 - ST B-L072Z-LRWAN1 LoRa®Discovery kit (with muRata radio chip).
+- IMST iM880B.
```

Comment:
Could you create a different list to indicate that this board was tested, but now by Arm?

Perhaps something like:
Here is a list of boards and modules that have been tested by the community:

Reply:
I have added a separated list.

@evedon , @AnttiKauppila : could you please review it again? Thanks!

Reply:
@evedon , @AnttiKauppila : could you please confirm if last update is ok? Thanks!